packages/fast-components-styles-msft/src/utilities/color/accent-fill.ts

```diff
@@ -1,92 +1,62 @@
 import { DesignSystem, DesignSystemResolver } from "../../design-system";
 import {
+    accentFillActiveDelta,
+    accentFillHoverDelta,
+    accentFillRestDelta,
+    accentFillSelectedDelta,
+} from "../design-system";
+import { accentSwatch, findAccessibleAccentSwatchIndexs } from "./accent";
+import { accentForegroundCut } from "./accent-foreground-cut";
+import {
+    colorRecipeFactory,
     FillSwatchFamily,
     Swatch,
     SwatchFamilyResolver,
     swatchFamilyToSwatchRecipeFactory,
     SwatchFamilyType,
     SwatchRecipe,
 } from "./common";
-import { accentSwatch, findAccessibleAccentSwatchIndexs } from "./accent";
 import { getSwatch, isDarkMode, Palette, palette, PaletteType } from "./palette";
-import { accentForegroundCut } from "./accent-foreground-cut";
-import {
-    accentFillActiveDelta,
-    accentFillHoverDelta,
-    accentFillRestDelta,
-    accentFillSelectedDelta,
-} from "../design-system";
 
-/**
- * Derives rest/hover/active active fill colors
- */
-export const accentFillAlgorithm: (
-    designSystem: DesignSystem,
-    contrastTarget: number
-) => FillSwatchFamily = (
-    designSystem: DesignSystem,
-    contrastTarget: number
-): FillSwatchFamily => {
-    const accentPalette: Palette = palette(PaletteType.accent)(designSystem);
-    const accent: Swatch = accentSwatch(designSystem);
-    const textColor: Swatch = accentForegroundCut(
-        Object.assign({}, designSystem, {
-            backgroundColor: accent,
-        })
-    );
-    const indexes: {
-        rest: number;
-        hover: number;
-        active: number;
-    } = findAccessibleAccentSwatchIndexs(designSystem, contrastTarget, textColor, {
-        rest: accentFillRestDelta(designSystem),
-        hover: accentFillHoverDelta(designSystem),
-        active: accentFillActiveDelta(designSystem),
-    });
+function accentFillAlgorithm(contrast: number): DesignSystemResolver<FillSwatchFamily> {
```

Comment:
@nicholasrice Is this breaking? Looks like the function is no longer exported.

Reply:
I don't think so - it is never exported from the package root, ie I cant do the following:
```
import { accentFillAlgorithm } @microsoft/fast-components-styles-msft
```